exchange/exchange.go

```diff
@@ -66,7 +66,8 @@ type exchange struct {
 // Container to pass out response ext data from the GetAllBids goroutines back into the main thread
 type seatResponseExtra struct {
 	ResponseTimeMillis int
-	Errors             []openrtb_ext.ExtBidderError
+	Errors             []openrtb_ext.ExtBidderMessage
+	Warnings           map[openrtb_ext.BidderName][]openrtb_ext.ExtBidderMessage
```

Comment:
There wouldn't be a need to add `Warnings` here. They would still be represented as `Errors` at this point.

Reply:
`openrtb_ext.ExtBidderMessage` doesn't support severity level. To consolidate Errors and Warnings we need to modify it and have `message`, `code` and `severity level` properties in it. We discussed this with @hhhjort here: https://github.com/prebid/prebid-server/pull/1724#discussion_r582575208